service/controller/v1/prometheus/prometheus.go

```diff
@@ -217,6 +241,8 @@ var (
 	// RelabelNamespaceRegexp is the regular expression to match against metrics with empty exported_namespace and namespace kube-system.
 	RelabelNamespaceRegexp = config.MustNewRegexp(`;(kube-system|giantswarm.*|vault-exporter)`)
 
+	ManagedAppSourceRegexp = config.MustNewRegexp(`(.*);(.*);(.*);(.*)`)
```

Comment:
What does this regex cover?

Reply:
We have 4 different sources from rules, https://github.com/giantswarm/prometheus-config-controller/blob/d9927cf11f3a6545e2adc4a7ccad780a2d6afa78/service/controller/v1/prometheus/scrapeconfig.go#L162

They need to be separated and using target labels. I'm not sure how to manage it a more elegant way to do this.

Reply:
@tuommaki Do you mean by `RelabelNamespaceRegexp`? That regex only works in `workload` job. 
But, yes. If customers put the annotations themselves, we can't stop them 😆

Reply:
Actually now that I double check, I think I got confused by the regex usage. The labels are there as mentioned, but that's not related to this regex. I think it's all good here. Ignore me. :slightly_smiling_face: